_opcodes/op187.html

```diff
@@ -1,6 +1,6 @@
 ---
 n: 187
-name: "Script: Store Local Variable"
+name: "Script: Set Local Variable"
```

Comment:
Why the rename?

Reply:
> Why the rename?

To better clarify that this opcode can only SET a value (op309 instead can SET and INCREMENT).

Reply:
I'm not sold on the idea — "store" doesn't imply any modification potential, while "set" does.

Reply:
> I'm not sold on the idea — "store" doesn't imply any modification potential, while "set" does.

OK, reverted back to "Store".